### DISCUSSION TITLE
[Resizable Column] How to resize columns by moving just one side not both left and right border

### DISCUSSION THREAD

**Discussion by 03aquamarine** [ant-design/ant-design#49111] [Q&A]
I'm using antd table and have added resizable column on the header.
When resizing the column, it seems like that both left and right border of each column is being moved though I just dragged the right border. It's quite bad for user experience.
**I want to move only the right border or at least just one side of a column not both.**
I've tried the demo [https://ant.design/components/table#components-table-demo-resizable-column](url) and it's what I expect.
I used the same code as the demo.
How can I resolve this problem?

This is the current state.
https://github.com/ant-design/ant-design/assets/126101776/a3143a29-5295-4d94-b7bd-86547c09ba02

This is what I want.
https://github.com/ant-design/ant-design/assets/126101776/b3f7259f-5a17-41cf-bc97-ed05a59a70ee

**Comment by linxianxi** [comment]
wait this https://github.com/react-component/table/pull/1106

**Reply by 03aquamarine** [reply]
Thanks for your comment.
I read the link you gave me, but I'm not sure yet.
I gave each column `fixed: "left"` attribute, but it only works more awkward.
or do you mean I should use `rc-table`?
I can't speak Chinese at all, so I'd appreciate it if you could explain it in detail.

**Reply by linxianxi** [reply]
you should update scroll.x

**Reply by 03aquamarine** [reply]
`scroll={{ x: 1000 }}` attribute is already applied to Table component.
How can I update this?

**Reply by ashishgupta-sf** [reply]
Did you get a work around this?
@03aquamarine

**Reply by 03aquamarine** [reply]
Sorry for late reply. I remember that I finished this problem by setting the minimum width and maximum width for each column. By the way, my project needed to change the grid library for other issue, so I solved it with ag-grid not antd-table.

**Comment by Naveenbc** [comment]
@ashishgupta-sf , @03aquamarine 
Did you get a work around this?

**Reply by 03aquamarine** [reply]
I remember that I finished this problem by setting the minimum width and maximum width for each column. By the way, my project needed to change the grid library for other issue, so I solved it with ag-grid not antd-table.
